src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseObj = {};
+  const repeatedVal = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (repeatedVal[value]) {
+      return null;
+    }
```

Comment:
you can check if there is repeatedVal by checking if `inverseObj` has this property already - you don't need to use another object to store that